allennlp/data/dataset_readers/snli.py

```diff
@@ -37,15 +35,15 @@ class SnliReader(DatasetReader):
     def __init__(self,
                  tokenizer: Tokenizer = None,
                  token_indexers: Dict[str, TokenIndexer] = None) -> None:
+        super().__init__()
```

Comment:
What's the general recommendation in python for using a bare `super()` versus `super(Class, self)`?  I learned to use the latter, so that's what I've always done.  Does it matter?

Reply:
not sure about general recommendation, the docs just use `super()`:

https://docs.python.org/3/library/functions.html#super

my understanding is that if you're inside a class method they're equivalent (that is, the `Class` is implicitly the current class), but if you're not inside a class method then you're required to be explicit (obviously)